sdk/cosmos/src/clients/collection_client.rs

```diff
@@ -46,12 +46,12 @@ impl CollectionClient {
     /// Get a collection
     pub async fn get_collection(
         &self,
-        ctx: Context,
         options: GetCollectionOptions,
     ) -> crate::Result<GetCollectionResponse> {
         let mut request = self.prepare_request_with_collection_name(http::Method::GET);
 
-        let mut pipeline_context = PipelineContext::new(ctx, ResourceType::Collections.into());
+        let mut pipeline_context =
+            PipelineContext::new(Context::new(), ResourceType::Collections.into());
```

Comment:
Maybe we can tackle this in a following PR but I think we can get rid of the `Context` here as well: `PipelineContext` is already enough (no need to instantiate an inner, useless `Context`).